src/views/Help/help.scss

```diff
@@ -1,7 +1,36 @@
 @import '../../vars';
 
-strong {
-  font-size: 15px;
-  font-weight: bold;
-  margin-top: 20px;
+
+
+.help {
+  
+  strong {
+    font-weight: bold;
+    margin-top: 20px;
+  }
+  
+  margin: 1rem;
+
+  li {
+    list-style-type: '⮞ ';
+  }
+
+  li li {
+    list-style-type: '⮚ ';
+  }
```

Comment:
Why do you have both of these?

Reply:
The second one is the indented list item. See attached photos under 'Home:'

Reply:
Oooh, I see now that those are different characters. I hate that we have to manually define the nesting styles for this case, but it makes sense at least.